gcloud-java-bigquery/src/main/java/com/google/gcloud/spi/DefaultBigQueryRpc.java

```diff
@@ -91,7 +92,14 @@ private static BigQueryException translate(IOException exception) {
 
   private static BigQueryException translate(GoogleJsonError exception) {
     boolean retryable = RETRYABLE_CODES.contains(exception.getCode());
-    return new BigQueryException(exception.getCode(), exception.getMessage(), retryable);
+    BigQueryError bigqueryError = null;
+    if (exception.getErrors() != null  && !exception.getErrors().isEmpty()) {
+      GoogleJsonError.ErrorInfo error = exception.getErrors().get(0);
```

Comment:
I think the api client had it as a list for flexibility but so far I am not aware of services using that and prefered
to keep the API simple in that case.

I think It should not be hard to nest the Exceptions later on if needed.